src/m_menu.c

```diff
@@ -6904,6 +6904,7 @@ void M_Init(void)
       ReadDef1.routine = M_DrawReadThis1;
       ReadDef1.x = 330;
       ReadDef1.y = 165;
+      HelpDef.y = 165;
```

Comment:
Does this also look alright with the dynamic help screen?

Reply:
Yes, we have redesigned the dynamic help screen:

![woof0001](https://user-images.githubusercontent.com/5077629/205703277-f4648aea-5659-4acd-90bb-620a64dd22eb.png)

Reply:
I know! 😁 I just couldn't imagine if the moved skull would still fit well.